serrano/resources/field/base.py

```diff
@@ -82,7 +85,14 @@ def get_link_templates(self, request):
         }
 
     def get_queryset(self, request, params):
-        queryset = self.model.objects.all()
+        tree = trees[params['tree']]
+        q = Q()
+
+        # No public method for accessing the local models on the tree
+        for app_name, model_name in tree._models:
+            q |= Q(app_name=app_name, model_name=model_name)
+
+        queryset = self.model.objects.filter(q)
```

Comment:
Doing an OR on the `DataConcept` filtering requires only one field to match the condition. A concept could be created that includes a field from an excluded or unrelated model, so the requirement is to exclude concepts if any fields do not match.